src/base/tz_support.c

```diff
@@ -5469,3 +5469,30 @@ conv_tz (void *p_out, const void *p_in, DB_TYPE type)
 
   return err_status;
 }
+
+/*
+ * tz_get_offset_in_mins () - time zone offset in minutes from GMT
+ */
+int
+tz_get_offset_in_mins ()
+{
+  time_t currtime;
+  struct tm *timeinfo;
+
+  time (&currtime);
+  timeinfo = gmtime (&currtime);
+  time_t utc = mktime (timeinfo);
+  timeinfo = localtime (&currtime);
+  time_t local = mktime (timeinfo);
+
+  // Get offset in minutes from UTC
+  double offsetFromUTC = difftime (utc, local) / 60;
+
+  // Adjust for DST
+  if (timeinfo->tm_isdst)
+    {
+      offsetFromUTC -= 60;
```

Comment:
I looked more into the code and I think best option is to change the implementation of TZ_OFFSET (https://www.cubrid.org/manual/en/11.0/sql/function/datetime_fn.html#tz-offset)
which is the only context for using db_sys_timezone function.
We should prefer using our TZ implementation to using system functions.
This should be a concern for another issue.

For this PR, I propose to not adjust for DST:
1) Old version of db_sys_timezone uses ftime
I didn't found a clear description of expected value of ftime->timezone regarding if it already includes dst offset or not; in the previous code, we don't take ftime->dstflag into account.
I suggest make a small test with ftime and check if timezone field already includes DST or not.
We should return the same value as by using ftime (have no test regression)

2) DST offset is not always -60 mins.
For instance :
https://github.com/CUBRID/cubrid/blob/develop/timezones/tzdata/asia#L1982
(20 minutes)